frontend/src/design/components/popovers/AccountPopover.js

```diff
@@ -12,10 +12,10 @@ import {
 import { useSnackbar } from 'notistack';
 import { useRef, useState } from 'react';
 import { Link as RouterLink, useNavigate } from 'react-router-dom';
-import { useAuth } from 'authentication';
 import { useGroups } from 'services';
 import { CogIcon } from '../../icons';
 import { TextAvatar } from '../TextAvatar';
+import { useAuth } from '../../../authentication';
```

Comment:
Better to use the global import (`import { useAuth } from 'authentication';`) instead of the relative import (`import { useAuth } from '../../../authentication';`)